desktop_version/src/Ent.h

```diff
@@ -5,6 +5,34 @@
 
 #define        rn( rx,  ry) ((rx) + ((ry) * 100))
 
+enum EntityTypes
```

Comment:
I'd personally call this `EntityType`, so that a variable of that type isn't an "entity types".